src/classes/LVL_LevelAssign_BATCH.cls

```diff
@@ -230,9 +239,17 @@ public without sharing class LVL_LevelAssign_BATCH implements Database.Batchable
     private class LevelException extends Exception {}
     
     /*********************************************************************************************************
-    * @description Batch process finish method, does nothing.
+    * @description Batch process finish method, stores last job start time for accounts and contacts
     */
-    public void finish(Database.BatchableContext BC) {}
+    public void finish(Database.BatchableContext BC) {
+        if (strObject == String.valueOf(Account.SObjectType)) {
+            levelsSettings.LastJobStartTimeAccount__c = batchStartTime;
```

Comment:
Do we need to verify the batch succeeded in the finish method?

Reply:
that's a good idea, I'll do a query on the batch job and check if the status is Completed before setting the fields